silabs-pti/build.gradle

```diff
@@ -28,7 +29,7 @@ java {
   withJavadocJar()
   withSourcesJar()
   toolchain {
-    languageVersion = JavaLanguageVersion.of(11)
+    languageVersion = JavaLanguageVersion.of(21)
```

Comment:
Don't use java 21 for PTI. This is included in many other places including customers using it directly. Don't force them to upgrade.